crates/re_types_builder/src/codegen/cpp/mod.rs

```diff
@@ -2345,7 +2345,7 @@ fn quote_arrow_field_type(
 ) -> TokenStream {
     let name = &field.name;
     let datatype = quote_arrow_data_type(&field.typ, objects, includes, false);
-    let is_nullable = field.is_nullable;
+    let is_nullable = field.is_nullable || field.typ == Type::Unit; // null type is always nullable
```

Comment:
shouldn't that be baked into `field.is_nullable` then?

Reply:
It is - also. I wen't for a belt-and-suspenders approach here :)